ruby/vampires.rb

```diff
@@ -0,0 +1,70 @@
+
+applicant_name = #{name}
+  puts "what is your name"
+  name = gets.chomp
+
+def age_correct
+  puts "what is your age"
+  age = gets.chomp.to_i
+  puts "what year were you born"
+  birth_year = gets.chomp.to_i
+if 2017-birth_year == age
+  age_correct = TRUE
+  puts "birth year approved"
+else
+  age_correct = FALSE
+  puts "birth year incorrect"
+
+end
+
+end
+age_correct
+
+def garlic_OK
+likes_garlic = #{garlic_preference}
+  puts "Would you like garlic bread? y/n"
+  garlic_preference = gets.chomp
+if garlic_preference == "y"
+  garlic_OK = TRUE
+else
+  garlic_OK = FALSE
+  puts "does not like garlic"
```

Comment:
The issue that you will run into here with this return is that `puts` is printing a string to the console and then returning `nil`.  puts returns nil and since this is the last line in your false section of the method you are actually not returning a value.

Give this a refactor and reach out to me with questions.